xwalk_android_tests.gypi

```diff
@@ -26,6 +26,17 @@
       'includes': [ '../build/java.gypi' ],
     },
     {
+      'target_name': 'native_echo_extension',
+      'type': 'loadable_module',
+      'include_dirs': [
+        '..',
+      ],
+      'sources': [
+        'test/android/data/nativeechoextension/native_echo_extension.c',
```

Comment:
I presume the content of native_echo_extension.c is same as `src/xwalk/extensions/test/echo_extension.c`. Can we re-use the the code and target instead of copy?